eng/pipelines/vmr-build.yml

```diff
@@ -28,6 +28,8 @@ parameters:
   default: false
 
 variables:
+- template: /eng/common/templates-official/variables/pool-providers.yml@self
```

Comment:
I think this file is not used anymore. Will verify

Reply:
Oh yeah, it can be removed. Will wait for the builds to go green first and push the removal